src/Transpiler.ts

```diff
@@ -305,7 +309,11 @@ export abstract class LuaTranspiler {
 
     public transpileLuaLibFunction(func: LuaLibFeature, ...params: string[]): string {
         this.importLuaLibFeature(func);
-        return `__TS__${func}(${params.join(", ")})`;
+        if (params.length > 1) {
+            return `__TS__${func}(${params.join(", ")})`;
+        } else {
+          return `__TS__${func}(${params[0]})`;
```

Comment:
What is the purpose of this if/else? `params[0]` and `params.join(", ")` look like they would produce exactly the same result?